pipelines/build-test-tool-template.yaml

```diff
@@ -1,45 +1,76 @@
 steps:
   - task: UseDotNet@2
-    displayName: 'Use .NET Core'
+    displayName: Use .NET Core
```

Comment:
Please revert changes in this file

Reply:
Why do you dislike these changes? Is it just because they're unrelated to the PR? They eliminated a bunch of warnings in the CI pipeline, which seems like a plus to me.

Reply:
I'm seeing a lot of commented code and unnecessary changes (like quotes and dotnet --info) which I don't like so it's hard to tell which of these changes are actually functional. Can you revert the unnecessary changes/ commented code at a minimum and I'll consider whatever's left over?

Reply:
I think it's about as small a change as I can make it. I considered combining restore/build or even restore/build/test, but there isn't much harm to keeping them separate.